src/test_mic.sh

```diff
@@ -45,8 +45,8 @@ Usage: $0 options
 
 OPTIONS:
 -h         Show this message
--i  <arg>  PCM input device name e.g. -i mic
--o  <arg>  PCM output device name e.g. -o hw:0,2
+-i  <arg>  PCM input device name e.g. -i default
+-o  <arg>  PCM output device name e.g. -o default
```

Comment:
since hw:0,2 can also work here, should we just have -o hw:0,2, default in the help for PCM output device?